assets/js/lib/model/clusters.js

```diff
@@ -16,6 +25,14 @@ const clusterTypeLabels = {
 export const getClusterTypeLabel = (type) =>
   clusterTypeLabels[type] || 'Unknown';
 
+const clusterScenarioLabels = {
+  [PERFORMANCE_SCENARIO]: 'Perf. Opt.',
+  [COST_OPT_SCENARIO]: 'Cost Opt.',
+};
+
+export const getClusterScenarioLabel = (type) =>
+  clusterScenarioLabels[type] || '';
```

Comment:
Do we need the `||`? Doesn't it work the same with just the access to the object?

Reply:
Fair enough :smile: 
For the future remember that we can also use `get` from lodash for better default semantics :+1: 

Thank you!